File: themes/OrchidEditorial/src/main/java/com/eden/orchid/editorial/EditorialModule.java
package com.eden.orchid.editorial;

import com.eden.orchid.api.registration.OrchidModule;
import com.eden.orchid.api.theme.Theme;

public class EditorialModule extends OrchidModule {

    @Override
    protected void configure() {
        addToSet(Theme.class, EditorialTheme.class);
    }
}

File: themes/OrchidEditorial/src/main/kotlin/com/eden/orchid/editorial/EditorialModule.kt
package com.eden.orchid.editorial

import com.eden.orchid.api.registration.OrchidModule
import com.eden.orchid.api.theme.Theme
import com.eden.orchid.utilities.addToSet

class EditorialModule : OrchidModule() {

    override fun configure() {
        addToSet<Theme, EditorialTheme>()
    }
}
